cmd/proxy/config/apiConfig/v1_0.toml

```diff
@@ -113,6 +113,6 @@ Routes = [
     { Name = "/:shard/raw/block/by-hash/:hash", Secured = false, Open = true, RateLimit = 0 },
     { Name = "/:shard/json/block/by-nonce/:nonce", Secured = false, Open = true, RateLimit = 0 },
     { Name = "/:shard/json/block/by-hash/:hash", Secured = false, Open = true, RateLimit = 0 },
-    { Name = "/:shard/raw/miniblock/by-hash/:hash", Secured = false, Open = true, RateLimit = 0 },
-    { Name = "/:shard/json/miniblock/by-hash/:hash", Secured = false, Open = true, RateLimit = 0 }
+    { Name = "/:shard/raw/miniblock/by-hash/:hash/epoch/:epoch", Secured = false, Open = true, RateLimit = 0 },
+    { Name = "/:shard/json/miniblock/by-hash/:hash/epoch/:epoch", Secured = false, Open = true, RateLimit = 0 }
```

Comment:
forgotten startofepoch endpoints?

Reply:
right. added them

Reply:
Forgotten `,` ? This looks like an invalid .toml file
Also, cmd/proxy/config/apiConfig/v_next.toml